### DISCUSSION TITLE
kurze Frage zum USR-TCP232-E2

### DISCUSSION THREAD

**Discussion by darkblu23** [Tom-Bom-badil/samson_trovis_557x#10] [General (de/en)]
Hallo Tom,
ich bin auf dein Wiki gestoßen, da ich auch eine Trovis 5573 habe. Diese würde ich auch gerne auslesen.
Ich habe mir jetzt den USR-TCP232-E2 bestellt, ist aber noch nicht da.
Du schreibst im Wiki:
"_Die Trovis über das angefertigte Kabel mit dem Adapter verbinden und das Netzwerkkabel in den Adapter stecken. Das Handy auf feste IP '192.168.0.111' einstellen und im Handy-Browser die '192.168.0.7' aufrufen_"
Demnach wird das eine Ende des selbstbau Kabels in die Trovis gesteckt und das andere Ende kommt an den USR-TCP232-E2 (an die Pin Leiste) ?
Muss der USR-TCP232-E2 dann per Netzwerkkabel an meine FritzBox angeschlossen werden, oder baut der USR-TCP232-E2 ein
WLAN auf ? Das ist mir nicht ganz klar.
Auf der Weboberfläche von dem USR-TCP232-E2 werden dann die Einstellungen vorgenommen wie auf deinen Screenshots, check.
Ist der USR-TCP232-E2 danach dann per WiFi erreichbar, oder muss er permanent per Kabel an der Fritzbox hängen ?

Sorry wenn ich dich hier frage, und Danke für deine Antwort ;-)

**Comment by Tom-Bom-badil** [comment]
Ja, das ist vielleicht ein bisschen missverständlich oder nicht ausführlich genug beschrieben.

Der E2 ist ein reiner LAN-Adapter, siehe Bilder hier im Wiki. Wird er an einen LAN-Port der Fritzbox angeschlossen, ist er trotzdem auch aus einem 'Nicht-Fritz-Netz' erreichbar, wenn das andere Endgerät im gleichen Subnetz wie der E2 ist. Also: E2=192.168.0.7 (default), Handy = 192.168.0.111 (manuell auf feste IP eingestellt), einbuchen des Handys in's WLAN, dann sollte der E2 unter seiner IP im Browser erreichbar sein.

Wenn alle Stricke reißen und es partout nicht gehen will - einen Notebook / PC kurz mal auf die .0.111er Adresse einstellen und mit dem E2 per Netzwerkkabel 'kurzschließen'. Die meisten modernen netzwerkadapter können heute automatisch zwischen Kreuzkabel und Parallelverdrahtung unterscheiden/umschalten.

Es gibt auch Adapter mit reinem WLAN (hab hier z.B. einen von ELFIN zu liegen) - bin aber nie dazu gekommen, die auch wirklich mal zu testen.

/tom

**Comment by darkblu23** [comment]
Hallo Tom, Danke für die prompte Antwort.
Das bringt auf jeden Fall Licht ins Dunkel.
Den Elfin EW 11 hab ich sogar hier, der ging sehr leicht ins WLAN, nur leider bekomme ich damit keine Verbindung zum Modbus Adapter im ioBroker (nur für 5 Sekunden, dann bricht die Verbindung ab und das im Minuten Takt)
Darum wollte ich das ganze jetzt mit dem USR-TCP232-E2 probieren.

Also nochmals herzlichen Dank

**Comment by calle2010** [comment]
Hallo @darkblu23, ich habe eine von deinem Thema abweichende Frage, wäre aber über eine kurze Antwort sehr glücklich: muss für den EW11 noch ein RS232/RS485 Adapter angeschlossen werden oder funktioniert der ohne weiteres an der 5573?

**Comment by darkblu23** [comment]
Hallöchen, alsooooo ich habe ihn ohne einen Zwischenstecker nach der TTL Methode angeschlossen (5573 ist ja TTL)
Tx auf A
Rx auf B
5V auf C, extern durch USB Ladegerät
GND auf D, extern durch USB Ladegerät
Da der Elfin etwas sendet und empfängt, würde ich deine Frage mit "ja" beantworten.
Allerdings frage ich mich, ob das Senden und Empfangen nicht genau andersrum sein müsste.
Leider sendet er mir nix in den ioBroker Modbus Adapter, vielleicht deswegen.
![SPS](https://github.com/Tom-Bom-badil/samson_trovis_557x/assets/149700663/361edf10-095d-4c21-b825-8c62729057d6)

**Comment by calle2010** [comment]
Danke. Ich denke, der serial port hätte auch ohne Drähte dran etwas gesendet. Auf der Ebene gibt es ja keinen Handshake.
Da aber auf dem serial port, laut Screenshot, nichts empfangen wurde gehe ich davon aus, dass das so wirklich nicht geht.
